lib/core/src/model.rs

```diff
@@ -833,6 +833,12 @@ pub struct BlockchainInfo {
     pub bitcoin_tip: u32,
 }
 
+#[derive(Copy, Clone)]
+pub(crate) struct ChainTips {
```

Comment:
Why not reuse `BlockchainInfo`?

Reply:
The concepts seemed different to me, even though currently they hold the same data. We may want to add non-tip-related data to `BlockchainInfo`. That said, I'm also fine with using the same struct for now and splitting only in the future if needed.